srml/executive/src/lib.rs

```diff
@@ -145,12 +149,16 @@ impl<
 	Block: traits::Block<Header=System::Header, Hash=System::Hash>,
 	Context: Default,
 	Payment: MakePayment<System::AccountId>,
+	Balance: SimpleArithmetic + Copy,
```

Comment:
Can't we use `BalanceOf<Payment, System::AccountId>` instead of additional generic type `Balance`?

Reply:
Yes indeed we actually don't need this anymore; taking your previous comment balance is now an associated type of make_payment; hence not really needed up here.